### ISSUE TITLE
Godot 4.0 support?

### ISSUE THREAD

**Issue by jon-heard** [timothyqiu/gdfxr#6]
I am curious about this addon's support for Godot 4.0.  I see that you have a branch for it, but it seems currently not to work.  Is this waiting on known Godot 4.0 issues to be resolved before it'll work?

Thanks for this useful plugin!

**Comment by timothyqiu** [comment]
I just tested with the latest 4.0.beta14 and it seems to work properly. What's the error you've encountered?

There's indeed a known issue though. If you click a slider to enter the value manually, you'll have to hit enter manually to exit (or click other buttons).

**Comment by jon-heard** [comment]
Sorry for not being specific.  I assumed since it's not in the add-on library that it was probably in mid-development.

The specific issue that I have is that I can load and play the SFXR files that I created in 3.5, and I can even modify those SFXR files and re-save them, but if I try to save a new SFXR file, the file fails to import.  It shows up with a red "X" icon and the console shows these errors:

```
  Unimplemented _get_priority in add-on.
  Unimplemented _get_priority in add-on.
  editor/editor_file_system.cpp:1942 - BUG: File queued for import, but can't be imported, importer for type '' not found.
  editor/editor_file_system.cpp:1943 - Method/function failed.
```

I know from experience that the first two aren't critical (though they are easy to fix).  The second two are likely directly attached to the issue.

In addition, once an SFXR file has failed to import in this way, subsequent files will also fail to import (seen with a 3d mesh importer addon and with the engine's default font importer).  It seems that there is a queue and the unimported SFXR file blocks the queue.

Also, if I duplicate a working SFXR file in my project, then the duplicate works ok, but if I copy/paste a working SFXR file in windows explorer, then Godot fails to import the copied file in the same way as described above.  I just discovered that Godot successfully duplicates SFXR files, which actually gives me a workaround for this issue.  That's nice.

I've seen this issue in my existing project (imported from 3.5), but I also just grabbed Godot 4 beta 14, made a fresh test project and saw the same issue.
